src/products/components/ProductVariantChannels/ChannelsAvailabilityCard/availabilityCount.test.ts

```diff
@@ -0,0 +1,200 @@
+import {
+  ChannelPriceAndPreorderData,
+  IChannelPriceAndPreorderArgs,
+} from "@dashboard/channels/utils";
+import { FormsetData } from "@dashboard/hooks/useFormset";
+import { product, variant } from "@dashboard/products/fixtures";
+
+import {
+  getAvailabilityCountForProduct,
+  getAvailabilityCountForVariant,
+} from "./availabilityCount";
+
+const variantFixture = variant("");
```

Comment:
Why do you use entire fixture for testing very little helper that obtains availability by using just a few fields? The fact that this helper use very wide type doesn't mean it should, the type could have been narrowed. 

When you make that simple test dependent on quite sophisticated fixture it may lead to complexity of maintenance for this test.

Reply:
Considering one of helper you uss, the thing you need is just `channelListings` and `product.channelListings` where each has field `channel` property with it's`id` - that's it:

```
channelListings: [{ channel: { id: "1" } }, { channel: { id: "2" } }, { channel: { id: "3" } }]
product: {
  channelListings: [{ channel: { id: "4" } }, { channel: { id: "3"  } }]
} 
```